components/validator/challenger.go

```diff
@@ -813,7 +813,7 @@ func (c *Challenger) CancelChallenge(ctx context.Context, outputIndex *big.Int)
 // ProveFault creates proveFault transaction for invalid output root.
 // TODO: ProveFault will take long time, so that we may have to handle it carefully.
 func (c *Challenger) ProveFault(ctx context.Context, outputIndex *big.Int, challenger common.Address, skipSelectFaultPosition bool) (*types.Transaction, error) {
-	c.log.Info("crafting proveFault tx")
+	c.log.Info("crafting proveFault tx", "outputIndex", outputIndex, "challenger", challenger)
```

Comment:
`ProveFault` is only called by challenger, do we need to log challenger address here?

Reply:
I think It's also informative as well which could be used related to their challenges.